components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/SiddhiAppStatus.java

```diff
@@ -52,7 +56,9 @@ public int getAge() {
     }
 
     public void setAge(int age) {
-        this.age = age;
+        //to make seconds
+        this.agetime = getTimeAgo(age);
+        this.age = age/1000;
```

Comment:
space after and before /

Reply:
Fixed in 68cfae165beb3c6758c81acd7bcf531a9636dc34